### ISSUE TITLE
Better tests

### ISSUE THREAD

**Issue by captn3m0** [hackercouch/hackercouch#35]
We can do so much more with travis:
1. Make sure that the couches are in `_couches` directory
2. Make sure all files have valid extensions
3. Anti-spam mechanisms via akismet. Run all the couch content for spam checks and fail the build if the spam scores are too high.
4. Validate the couch structure itself (required/optional fields)

**Comment by ppaeps** [comment]
On 2015-12-01 01:24:06 (+0530), Nemo notifications@github.com wrote:

> We can do so much more with travis:
> 1. Make sure that the couches are in `_couches` directory
> 2. Make sure all files have valid extensions

I've made a start on _2_ in pull request #36.  Not sure how to go about 
_1_.  Is this travis contraption like a Git `pre-receive` hook?  Just 
adding a check for new files outside `_couches` is probably a lot too 
naive. ;)

**Comment by captn3m0** [comment]
Yes, travis just runs the cibuild script and fails the build if it returns non-zero.

Yeah, checking outside couches seems a bit excessive. I should pay a bit more attention before clicking the Merge button, thats all.
